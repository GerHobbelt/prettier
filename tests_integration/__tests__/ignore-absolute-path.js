"use strict";

const path = require("path");
const runPrettier = require("../runPrettier");

test("support absolute filename", () => {
  const result = runPrettier("cli/ignore-absolute-path", [
    path.resolve(__dirname, "../cli/ignore-absolute-path/ignored/module.js"),
    path.resolve(__dirname, "../cli/ignore-absolute-path/depth1/ignored/*.js"),
    path.resolve(__dirname, "../cli/ignore-absolute-path/regular-module.js"),
    "-l"
  ]);

  expect(result.stdout).toMatchSnapshot();
  expect(result.status).toEqual(1);
});
