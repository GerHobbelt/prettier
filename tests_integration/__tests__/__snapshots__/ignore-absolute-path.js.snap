// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`support absolute filename 1`] = `
"regular-module.js
"
`;
